src/poke_env/player/utils.py

```diff
@@ -21,16 +21,21 @@
 
 def background_cross_evaluate(
     players: List[Player], n_challenges: int
-) -> "Future[Dict[str, Dict[str, Optional[float]]]]":  # pragma: no cover
+) -> "Future[Dict[str, Dict[str, Optional[float]]]]":
     return asyncio.run_coroutine_threadsafe(
         cross_evaluate(players, n_challenges), POKE_LOOP
     )
 
 
+PlayerType = TypeVar("PlayerType", bound=Player)
+
+
 async def cross_evaluate(
-    players: List[Player], n_challenges: int
+    players: List[PlayerType], n_challenges: int
```

Comment:
Why not `List[Player]`?

Reply:
I remember this being necessary at some intermediate step in the changeset, but you're right, it's not necessary now. Sorry about that: 287fcba6ba117d0a058ca744dca3dd97045e71e1